tests/openvino/native/quantization/test_fq_params_calculation.py

```diff
@@ -71,10 +71,10 @@ def compare_stats(expected, actual):
 
 
 # pylint: disable=protected-access
-def quantize_model(ov_model, preset):
+def quantize_model(ov_model, q_params):
```

Comment:
What is the profit of these changes?

Reply:
We now can add all necessary quantization parameters not only preset. For example, I used `model_type`